sla_per_user_system_test.py

```diff
@@ -622,3 +685,99 @@ def _throughput_latency_tests_run(self, read_cmds, read_users, latency_user):
             self.log.info(result_print_str)
         finally:
             self.clean_auth(entities_list_of_dict=read_users)
+
+    def _compare_workloads_c_s_metrics(self, workloads_queue: list) -> dict:
+        comparison_axis = {"latency 95th percentile": 2.0,
+                           "latency 99th percentile": 2.0,
+                           "op rate": 2.0}
+        workloads_results = {}
+        for workload in workloads_queue:
+            result = self.get_stress_results(queue=workload, store_results=False)
+
+            workloads_results.update({result[0].get("username"): result[0]})
+
+        assert len(workloads_results) == 2, "Got more / less than two c-s result:\n%s" % workloads_results
+        comparison_results = {}
+        try:
+            for item, target_margin in comparison_axis.items():
+                interactive = float(workloads_results["interactive"][item])
+                batch = float(workloads_results["batch"][item])
+                ratio = interactive / batch if item == "op rate" else batch / interactive
+
+                comparison_results.update(
+                    {
+                        item: {
+                            "interactive": interactive,
+                            "batch": batch,
+                            "diff": batch - interactive,
+                            "ratio": ratio,
+                            "within_margin": ratio >= target_margin
+                        }
+                    }
+                )
+            return comparison_results
+        except Exception:
+            logger.info("Failed to compare c-s results for batch and interactive"
+                        "workloads.")
+            raise
+
+    def upload_c_s_comparison_to_es(self) -> None:
+        logger.info("Uploading c-s comparison to ES...")
+        es_body = {
+            self._get_db_node_instance_type(): {
+                "test_id": self.test_id,
+                "backend": self.db_cluster.params.get("cluster_backend"),
+                "scylla_version": self.get_scylla_versions(),
+                **self._comparison_results
+            }
+        }
+        self._es.create_doc(index="workload_types", doc_type="test_stats",
+                            doc_id=self.test_id, body=es_body)
+        logger.info("C-s comparison uploaded to ES.")
+
+    def _get_db_node_instance_type(self):
+        backend = self.db_cluster.params.get("cluster_backend")
```

Comment:
What about docker?

Reply:
Not sure we're ever going to run this with docker, but just to be sure I added docker and azure here as well.